### ISSUE TITLE
nil pointer with kubernetes ingress

### ISSUE THREAD

**Issue by Hades32** [containous/traefik#934]
Traefik v1.1.1 Kubernets 1.4.6

Log:
```
time="2016-12-04T22:30:59Z" level=error msg="Error in Go routine: runtime error: invalid memory address or nil pointer dereference"
goroutine 34 [running]:
runtime/debug.Stack(0x0, 0x0, 0x0)
        /usr/local/go/src/runtime/debug/stack.go:24 +0x74
runtime/debug.PrintStack()
        /usr/local/go/src/runtime/debug/stack.go:16 +0x18
github.com/containous/traefik/safe.defaultRecoverGoroutine(0x99af60, 0x4420016020)
        /go/src/github.com/containous/traefik/safe/routine.go:135 +0x70
github.com/containous/traefik/safe.GoWithRecover.func1.1(0xb69740)
        /go/src/github.com/containous/traefik/safe/routine.go:126 +0x4c
panic(0x99af60, 0x4420016020)
        /usr/local/go/src/runtime/panic.go:458 +0x4c0
github.com/containous/traefik/provider.(*Kubernetes).loadIngresses(0x442030d400, 0xf4c3c0, 0x442044e1c0, 0x1, 0x0, 0x0)
        /go/src/github.com/containous/traefik/provider/kubernetes.go:199 +0x378
github.com/containous/traefik/provider.(*Kubernetes).Provide.func1.1(0x0, 0x0)
        /go/src/github.com/containous/traefik/provider/kubernetes.go:131 +0x4e8
github.com/containous/traefik/vendor/github.com/cenk/backoff.RetryNotify(0x4420448750, 0xf47880, 0x44204246b0, 0xb696e0, 0x0, 0x0)
        /go/src/github.com/containous/traefik/vendor/github.com/cenk/backoff/retry.go:32 +0x48
github.com/containous/traefik/provider.(*Kubernetes).Provide.func1(0x4420434150)
        /go/src/github.com/containous/traefik/provider/kubernetes.go:153 +0x188
github.com/containous/traefik/safe.(*Pool).Go.func1()
        /go/src/github.com/containous/traefik/safe/routine.go:73 +0x44
github.com/containous/traefik/safe.GoWithRecover.func1(0xb69740, 0x4420448150)
        /go/src/github.com/containous/traefik/safe/routine.go:129 +0x5c
created by github.com/containous/traefik/safe.GoWithRecover
        /go/src/github.com/containous/traefik/safe/routine.go:130 +0x3c
```

Ingress.yaml
``` 
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: v2c-path-ingress
#  annotations:
#    traefik.frontend.rule.type: pathprefix
spec:
  rules:
#  - host: k8s.local
   - http:
      paths:
      - path: /v2c
        backend:
          serviceName: v2c
          servicePort: http
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: v2c-host-ingress
spec:
  rules:
   - host: v2c.my.secret-host.de
   - http:
      paths:
      - backend:
          serviceName: v2c
          servicePort: http
```

**Comment by Hades32** [comment]
I had to kill my traefik pod to see any k8s updates after this...

**Comment by emilevauge** [comment]
Fixed by #944
